Examples/Neutral_density_calculation.ipynb

```diff
@@ -0,0 +1,790 @@
+{
```

Comment:
<p>Could we then have another example after the plots to do the same with <code>'mom6'</code> output? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/417/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>